### PR TITLE
Blueprint cleanup

### PR DESCRIPTION
**Pull request by Turbo87**
/cc @alexlafroscia

### CODE REVIEW

File: blueprints/ember-cli-eslint/index.js [hunk 1]
```diff
@@ -33,13 +33,13 @@ module.exports = {
   },
 
   afterInstall: function() {
-    var removeJSHintConfig = this._removeJSHintConfig.bind(this);
+    var removeJSHintDep = (this.removePackageFromProject && 'ember-cli-jshint' in this.project.dependencies()) ?
+      this.removePackageFromProject('ember-cli-jshint') :
+      Promise.resolve();
```
Comment (cibernox) [inline blueprints/ember-cli-eslint/index.js]: why this promise floating in the void?

Comment (rwjblue) [inline blueprints/ember-cli-eslint/index.js]: Ya, I think an if is likely better here for readability.

**Comment by rwjblue** [comment]
Minor personal nit-pick RE: if instead of ternary, but seems fine to land either way. 

:+1:

**Comment by Turbo87** [comment]
released as v2.0.1

**Pull request merged**